tests/components/zwave_js/test_api.py

```diff
@@ -2816,15 +2816,18 @@ async def test_firmware_upload_view(
     device = get_device(hass, multisensor_6)
     with patch(
         "homeassistant.components.zwave_js.api.begin_firmware_update",
-    ) as mock_cmd:
+    ) as mock_cmd, patch(
+        "homeassistant.components.zwave_js.api.USER_AGENT",
+        new_callable=PropertyMock(return_value={"HomeAssistant": "0.0.0"}),
```

Comment:
Why do we use a `PropertyMock` and not just a dict as `new` parameter? Or maybe use `patch.dict`?

Reply:
because I didn't know I could do either of those things 🙂